R/safetyGraphicsApp.R

```diff
@@ -1,33 +1,70 @@
 #' Run the interactive safety graphics builder
 #'
-#' @param charts Character vector of charts to include 
+#' @param charts Character vector of charts to include
 #' @param maxFileSize maximum file size in MB allowed for file upload
-#' @param settingsLocation folder location of user-defined settings metadata
+#' @param settingsLocation folder location of user-defined settings metadata. Files should be named settingsMetadata.rda, chartsMetadata.rda and standardsMetadata.rda and use the same structure established in the /data folder.
```

Comment:
are we going with rda or rds? Code uses Rds files but R packages store data as Rda.

Reply:
No strong opinion really. I guess the strength of RDS is that the files are smaller ... I'm fine with changing it if anyone feels strongly. Go ahead and file an issue if you think we should change to rda.